app/src/main/java/com/hedvig/app/feature/profile/ui/ProfileFragment.kt

```diff
@@ -43,6 +37,11 @@ class ProfileFragment : BaseTabFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        profileRoot.updatePadding(top = getToolbarBarHeight(this))
```

Comment:
We could remove synthetics from this file, but you should not as I am remaking the entirety of the `ProfileFragment` in another branch.

So yeah, don't do anything in particular to this file 😬